web/src/beta/lib/core/Crust/Widgets/useWidgetAlignSystem.ts

```diff
@@ -1,15 +1,27 @@
 import { useEffect, useState, useCallback } from "react";
 
-import type {
-  WidgetAlignSystem,
-  InternalWidget,
-  WidgetLocationOptions,
-  WidgetArea,
-  WidgetSection,
-  WidgetZone,
+import { isInvisibleBuiltin } from "./WidgetAlignSystem/utils";
+
+import {
+  type WidgetAlignSystem,
+  type InternalWidget,
+  type WidgetLocationOptions,
+  type WidgetArea,
+  type WidgetSection,
+  type WidgetZone,
 } from ".";
 
-export default ({ alignSystem }: { alignSystem: WidgetAlignSystem | undefined }) => {
+const sections: (keyof WidgetZone)[] = ["left", "center", "right"];
+const areas: (keyof WidgetSection)[] = ["top", "middle", "bottom"];
+const zones: (keyof WidgetAlignSystem)[] = ["outer", "inner"];
```

Comment:
As a rule (both in core and in reearth), can we start putting constants in a `constants.ts` file, uppercasing the names with underscores (ie. SECTIONS, DEFAULT_BACKGROUND_COLOR, etc) 🙏🏻

Reply:
Updated!